scripts/liminal

```diff
@@ -75,18 +75,21 @@ def deploy_liminal_core_internal(clean):
         print(f'Liminal was installed locally, changing the LIMINAL_VERSION parameter to {full_path}')
         os.environ[environment.LIMINAL_VERSION_PARAM_NAME]= full_path
     if clean:
-        docker_compose_command('down', ['--remove-orphans','--rmi', 'local'])
-        docker_compose_command('build', ['--no-cache'])
-        docker_compose_command('run', ['webserver', 'resetdb', '-y'])
-        docker_compose_command('run', ['webserver','initdb'])
-        docker_compose_command('down', ['--remove-orphans'])
+        docker_compose_command('call', 'down', ['--remove-orphans','--rmi', 'local'])
+        docker_compose_command('call', 'build', ['--no-cache'])
+        docker_compose_command('call', 'run', ['webserver', 'resetdb', '-y'])
+        docker_compose_command('call', 'run', ['webserver','initdb'])
+        docker_compose_command('call', 'down', ['--remove-orphans'])
 
-def docker_compose_command(command_name, args):
+def docker_compose_command(process, command_name, args):
     docker_compose_path, project_dir = get_docker_compose_paths()
     concated_args = ' '.join(args)
-    subprocess.call(
-        [f'docker-compose -f "{docker_compose_path}" -p liminal --project-directory {project_dir} {command_name} {concated_args}'],
-        env=os.environ, shell=True)
+    run_command = [f'docker-compose -f "{docker_compose_path}" -p liminal --project-directory {project_dir} {command_name} {concated_args}']
+    if 'call' in process:
```

Comment:
please use subprocess.run
which returns a CompletedProcess object with both std out, std err and return value.
for the one place which needs it (checking if liminal is running), you can retrieve STDOUT from this object and check if it contains liminal.

other invocations of this method anyhow don't use the retrun value